Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -519,7 +521,7 @@ namespace OpenInfraPlatform {
 						// convert the control points
 						std::vector<carve::geom::vector<3>> splinePoints = convertIfcCartesianPointVector(bspline_curve->ControlPointsList);
 
-						SplineConverterT<typename IfcEntityTypesT>::convertIfcBSplineCurve(bspline_curve, splinePoints, targetVec);
+						splineConverter->convertIfcBSplineCurve(bspline_curve, splinePoints, targetVec);

```

Comment:
```suggestion
						SplineConverterT splineConverterT(GeomSettings(), UnitConvert());
						splineConverter->convertIfcBSplineCurve(bspline_curve, splinePoints, targetVec);
```
Then there is no need for a member variable.

Reply:
Done in commit c09bd83